internal/cmd/asyncapi/command_import.go

```diff
@@ -42,13 +41,13 @@ type flagsImport struct {
 }
 
 type kafkaBinding struct {
-	BindingVersion string            `yaml:"bindingVersion"`
-	Partitions     int32             `yaml:"partitions"`
-	TopicConfigs   topicConfigPtr    `yaml:"topicConfiguration"`
-	XConfigs       map[string]string `yaml:"x-configs"`
+	BindingVersion     string                `yaml:"bindingVersion"`
+	Partitions         int32                 `yaml:"partitions"`
+	TopicConfiguration topicConfigurationPtr `yaml:"topicConfiguration"`
+	XConfigs           map[string]string     `yaml:"x-configs"`
 }
 
-type topicConfigPtr struct {
+type topicConfigurationPtr struct {
```

Comment:
Why does this end with "ptr"? The `type` itself is not a pointer, so is it named this way because it contains pointers?

Reply:
@brianstrauch topicConfiguration is used in command_export, but cant reuse because the fields have to be pointers here (nil when empty instead of default 0, they can define it to 0 cant tell difference)

not sure on a good name, topicConfigurationImport? haha

Reply:
> not sure on a good name, topicConfigurationImport?

Sure! `topicConfigurationImport` and `topicConfigurationExport` could be good for helping to differentiate between the two.